app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -35,22 +39,26 @@ public void setTitle(String title) {
         this.title = title;
     }
 
+    public List<Category> getCategories() {
```

Comment:
Yes. I think we should leave that until there is a clear need for it. I have not decided ye if a contest object can set its categories or if the builder object will set it.